x-pack/plugins/fleet/server/errors/handlers.ts

```diff
@@ -60,6 +61,9 @@ const getHTTPResponseCode = (error: IngestManagerError): number => {
   if (error instanceof PackageUnsupportedMediaTypeError) {
     return 415; // Unsupported Media Type
   }
+  if (error instanceof PackageFailedVerificationError) {
+    return 422; // Unprocessable Entity
```

Comment:
Not sure about this response code for whe na package fails verification, I was also tempted by Forbidden or Bad Request